src/lib/keychain/insecure.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import Configstore from 'configstore';
+
+/**
+ * Internal dependencies
+ */
+import type { Keychain } from './keychain';
+
+export default class Insecure implements Keychain {
+	private file: string;
+	private configstore: Configstore;
+
+	constructor( file: string ) {
+		this.file = file;
+
+		this.configstore = new Configstore( this.file );
+	}
+
+	getPassword( service: string ): Promise<string | null> {
+		try {
+			const value: unknown = this.configstore.get( service );
+			// eslint-disable-next-line @typescript-eslint/no-base-to-string
+			return Promise.resolve( null === value || undefined === value ? null : value.toString() ); // NOSONAR
```

Comment:
Could we keep this one closer to the original please? More readable but less compact.

Reply:
I have changed it, but I don't think it is more readable now.

The problem is that `get()` returns `any`, and we need a `string` or `null`. `undefined` and `null` don't have a `toString()` method, we have to filter them out. For the rest, we call `toString()` to get a string value. We cannot use template literals there because of "Invalid type "{}" of template literal expression."